static/js/enketo/cookies.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function() {
+  var cookies = {};
+
+  document.cookie.split(';')
+    .forEach(function( s ) {
+      var c = s.trim().split('=');
+      var key = c[0];
+      var val = c[1];
+      cookies[key] = val;
+    });
+
+  return cookies;
+};
```

Comment:
This is ok, but I think it'd be a little nicer to use the `Language` service which does all this for you. The downside is it returns a promise so the code would be async but I think that's ok.

If you disagree feel free to overrule me and merge anyway...